README.md

```diff
@@ -9,6 +9,7 @@ A curated list of awesome content created by Facebook Developer Circles communit
 - [Workshops](#workshops)
 - [Articles](#articles)
 - [Tools](#tools)
+- [First Contributions](#contributions)
```

Comment:
I think that the title need to be more clear and explicit! I find it a little confusing for 2 main reasons:
+ You're including only FB OSS repos, the title should state it clearly.
+ When I read it I first thought it's for this repo!

Thanks

Reply:
@Alquimista3301 I like your suggestions on adding more category to the list. Can you please amend the PR based on the suggestion from @aboullaite before we merge this? Thanks!